spec/views/cases/show_html_slim_spec.rb

```diff
@@ -310,4 +313,62 @@ def setup_policies(policies)
       end
     end
   end
+
+  describe 'SAR case' do
+    context 'before it is extended' do
+      let(:sar) { create(:approved_sar).decorate }
+
+      before do
+        assign(:case, sar)
+        assign(:permitted_events, [:extend_sar_deadline])
```

Comment:
does this need to be in both the permitted events and filtered permitted events?

Reply:
Yes in `cases/show.html.slim` the action bar only shows if `@filtered_permitted_events` has values and then only outputs buttons if `@permitted_events` is also populated.